config.h.in

```diff
@@ -0,0 +1,13 @@
+// the configured options and settings 
+// do not edit this file
```

Comment:
Oh, no... Sorry to rain on your parade, but I used libraries that generated source files using the buildsystem. It was always a mess. Also, I don't see the reason for this. It only serves to make things complicated.

Reply:
I'm not stupid

Reply:
What in my comment makes you say that?